EventFilter/SiPixelRawToDigi/src/PixelUnpackingRegions.cc

```diff
@@ -21,22 +21,22 @@
 #include <iterator>
 
 // local convenience functions
-namespace 
-{
-bool isBPIXFED(unsigned int fed) {return fed< 32;}
-bool isFPIXFED(unsigned int fed) {return fed>=32;}
-bool isBPIXModule(unsigned int id) {return DetId(id).subdetId() == PixelSubdetector::PixelBarrel;}
-bool isFPIXModule(unsigned int id) {return DetId(id).subdetId() == PixelSubdetector::PixelEndcap;}
-
-inline std::ostream& operator<<(std::ostream& s, const PixelUnpackingRegions::Module& m)
-{
-  s<< (isBPIXModule(m.id) ? "BPIX " : "FPIX ") <<m.id<<" "<<m.fed<<"   "<<m.phi<<"   "<<m.x<<" "<<m.y<<" "<<m.z<<"  "<<sqrt(std::pow(m.x,2)+std::pow(m.y,2));
-  return s;
-}
+namespace {
+  bool isBPIXFED(unsigned int fed) {return fed< 32;}
+  bool isFPIXFED(unsigned int fed) {return fed>=32;}
+  //bool isBPIXFEDPhase1(unsigned int fed) {return fed< 1276;}
+  //bool isFPIXFEDPhase1(unsigned int fed) {return fed>=1276;}
```

Comment:
is this commented out code needed?
Please remove or add comments inline in the code why the commented out block is relevant